code/src/pages/product-details/components/product-details-container/ProductDetailsContainer.tsx

```diff
@@ -119,9 +119,9 @@ const ProductDetailsContainer = ({
               >
                 {formatPrice(product.price)}
               </AppTypography>
-              <AppButton>
-                <AppTypography translationKey="productDetailsPage.buyNowButton" />
-              </AppButton>
+              <BuyNowButton
+                productWithId={{ ...product, id: productId }}
```

Comment:
Maybe would be better to move it above the return? What do you think?

Reply:
Thanks, I fixed it